wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -127,7 +137,7 @@ public function update( $new_instance, $instance ) {
 	 * @return void
 	 */
 	public function render_media( $instance ) {
-		$instance = array_merge( $this->default_instance, $instance );
+		$instance = array_merge( wp_list_pluck( $this->get_instance_schema(), 'default' ), $instance );
```

Comment:
Because it's part of the format for JSON Schema used in the REST API, anticipating a future move toward exposing these widgets in the REST API. Also, we avoid having to copy the keys down twice if we organize them under one  schema array instead of two separate arrays. Anyway, I think `wp_list_pluck` is fast so I'm not concerned about optimizing for it.